commands/update.js

```diff
@@ -0,0 +1,58 @@
+const exec = require('child_process').exec;
```

Comment:
Is there a programmatic way to use NPM? Calling the NPM CLI will require stdout text processing which isn't as clean as an API.